### PR TITLE
add values checking to mft test

### PR DESCRIPTION
**Pull request by nllong**
#### Any background context you want to provide?

We haven't been checking the results of the simulations.

#### What does this PR accomplish?

Check the results of the MFT model

#### How should this be manually tested?

* py.test

#### What are the relevant tickets?

#303

### CODE REVIEW

File: tests/model_connectors/test_district_heating_and_cooling_systems.py [hunk 1]
```diff
@@ -141,7 +141,8 @@ def test_district_heating_and_cooling_systems(self):
         # check the mass flow rates of the first load are in the expected range
         load = loads[0]
         (_, heat_m_flow) = mat_results.values(f'{load.id}.ports_aHeaWat[1].m_flow')
-        (_, cool_m_flow) = mat_results.values(f'{load.id}.ports_aHeaWat[1].m_flow')
+        # NL: changed the line below to be aChiWat (not repeating aHeaWat).
```
Comment (macintoshpie) [inline tests/model_connectors/test_district_heating_and_cooling_systems.py:144]: 😬 great catch

File: tests/model_connectors/test_time_series_mft_ets.py [hunk 2]
```diff
@@ -113,6 +116,62 @@ def test_mft_time_series_to_modelica_and_run(self):
         district.to_modelica()
 
         root_path = os.path.abspath(os.path.join(district._scaffold.districts_path.files_dir))
-        self.run_and_assert_in_docker(os.path.join(root_path, 'DistrictEnergySystem.mo'),
+        mo_file_name = os.path.join(root_path, 'DistrictEnergySystem.mo')
+        # set the run time to 31536000 (full year in seconds)
+        mofile = Model(mo_file_name)
+        mofile.update_model_annotation({"experiment": {"StopTime": 31536000}})
```
Comment (macintoshpie) [inline tests/model_connectors/test_time_series_mft_ets.py:122]: maybe we should make runtime templatable later?

**Review by macintoshpie — APPROVED** [review]
Looks great 👍 👍

File: tests/model_connectors/test_time_series_mft_ets.py (hunk 2 above)
Comment (nllong) [reply tests/model_connectors/test_time_series_mft_ets.py:122]: totally, and some helper functions like `mofile.set_stoptime(time.annual_in_seconds)`.

**Pull request merged**